lib/api/BackbeatAPI.js

```diff
@@ -527,6 +535,48 @@ class BackbeatAPI {
         return { reqBody };
     }
 
+    /**
+     * Pause CRR operations for given site(s)
+     * @param {Object} details - The route details
+     * @param {String} body - The POST request body string
+     * @param {Function} cb - The callback to call
+     * @return {undefined}
+     */
+    pauseCRRService(details, body, cb) {
+        let sites;
+        if (details.site === 'all') {
+            sites = details.extensions.crr.filter(s => s !== 'all');
+        } else {
+            sites = [details.site];
+        }
+        sites.map(s => `${this._crrTopic}-${s}`).forEach(channel => {
```

Comment:
No need to use `map` and create a temporary array, just do `sites.forEach(site => {...})`